{{cookiecutter.github_repository}}/pyproject.toml

```diff
@@ -78,6 +78,7 @@ django-mail-templated = "^2.6"
 # Static Files and Media Storage
 # -------------------------------------
 gunicorn = "~20.1.0"
+uvicorn = "^0.21.0"
```

Comment:
Should this be always installed? I would imagine it's either gunicorn or this?

Reply:
For dev/local, uvicorn is definitely needed.
For Production, it's recommended to use [Gunicorn as a process manager with the Uvicorn worker class](https://docs.djangoproject.com/en/4.1/howto/deployment/asgi/uvicorn/#deploying-django-using-uvicorn-and-gunicorn) but you're right, only gunicorn could also do the job ([comes with greenlets preinstalled](https://docs.gunicorn.org/en/stable/design.html#async-workers)) but it's an alternative to Python's built in coroutines and does require us to "monkey patch" applications.

Reply:
Alternatively, we can provide cookiecutter option "use_asgi". When not opted, we don't use uvicorn.

Reply:
`use_asgi` sounds good to me.